e2e-tests/commands/insurance/complete-how-much-the-agent-is-charging-form.js

```diff
@@ -12,7 +12,9 @@ const {
  * @param {String} fixedSumAmount: Fixed sum amount
  */
 const completeHowMuchTheAgentIsChargingForm = ({ fixedSumAmount = application.EXPORT_CONTRACT.AGENT_CHARGES[FIXED_SUM_AMOUNT] }) => {
-  cy.keyboardInput(field(FIXED_SUM_AMOUNT).input(), fixedSumAmount);
+  if (fixedSumAmount) {
```

Comment:
do we need the if statement if this function doesn't do anything else? 
Maybe completeHowMuchTheAgentIsChargingForm should only be called if fixedSumAmount

Reply:
Good shout - this was for submitting a form via "Save and back" without entering anything. I've simplified the test and removed the condition from the command.